packages/quantic/force-app/main/default/lwc/quanticRadioButtonsGroup/quanticRadioButtonsGroup.js

```diff
@@ -0,0 +1,168 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticRadioButtonsGroupOption` type defines the shape of an option
+ * that can be passed to the `QuanticRadioButtonsGroup` component.
+ * @typedef {Object} QuanticRadioButtonsGroupOption
+ * @property {String} value The value of the option.
+ * @property {String} [label] The label of the option.
+ * @property {String} [iconName] The name of the icon to be displayed.
+ * @property {String} [tooltip] The tooltip of the option.
+ */
+
+/**
+ * The `QuanticRadioButtonsGroup` component displays a group of radio buttons.
+ * @category Internal
+ * @example
+ * <c-quantic-radio-buttons-group options={options} legend="my legend"></c-quantic-radio-buttons-group>
+ */
+export default class QuanticRadioButtonsGroup extends LightningElement {
+  /**
+   * The value of the label for the Radio Buttons Group.
+   * @api
+   * @type {String}
+   */
+  @api legend = '';
+  /**
+   * The list of options to be displayed.
+   * @api
+   * @type {QuanticRadioButtonsGroupOption[]}
+   */
+  @api options;
+  /**
+   * If the labels should be hidden or visible.
```

Comment:
```suggestion
   * Whether the labels should be hidden.
```

It's a total pet peeve of mine, though, so feel free to ignore 😂

Reply:
I think mine is a lot easier to read quickly lol :P